tuf/client/updater.py

```diff
@@ -1561,85 +1659,73 @@ def _update_metadata_if_changed(self, metadata_role,
 
 
 
-  def _fileinfo_has_changed(self, metadata_filename, new_fileinfo):
+  def _versioninfo_has_changed(self, metadata_filename, new_versioninfo):
     """
     <Purpose>
-      Non-public method that determines whether the current fileinfo of
-      'metadata_filename' differs from 'new_fileinfo'.  The 'new_fileinfo'
+      Non-public method that determines whether the current versioninfo of
+      'metadata_filename' differs from 'new_versioninfo'.  The 'new_versioninfo'
       argument should be extracted from the latest copy of the metadata that
       references 'metadata_filename'.  Example: 'root.json' would be referenced
       by 'snapshot.json'.
         
-      'new_fileinfo' should only be 'None' if this is for updating 'root.json'
-      without having 'snapshot.json' available.
+      'new_versioninfo' should only be 'None' if this is for updating
+      'root.json' without having 'snapshot.json' available.
 
     <Arguments>
       metadadata_filename:
         The metadata filename for the role.  For the 'root' role,
         'metadata_filename' would be 'root.json'.
 
-      new_fileinfo:
+      new_versioninfo:
         A dict object representing the new file information for
-        'metadata_filename'.  'new_fileinfo' may be 'None' when
+        'metadata_filename'.  'new_versioninfo' may be 'None' when
         updating 'root' without having 'snapshot' available.  This
-        dict conforms to 'tuf.formats.FILEINFO_SCHEMA' and has
+        dict conforms to 'tuf.formats.VERSIONINFO_SCHEMA' and has
         the form:
-        {'length': 23423
-         'hashes': {'sha256': adfbc32343..}}
+        
+        {'version': 288}
         
     <Exceptions>
       None.
 
     <Side Effects>
-      If there is no fileinfo currently loaded for 'metada_filename',
+      If there is no versioninfo currently loaded for 'metada_filename',
       try to load it.
 
     <Returns>
-      Boolean.  True if the fileinfo has changed, false otherwise.
+      Boolean.  True if the versioninfo has changed, false otherwise.
```

Comment:
1: "[...], *False* otherwise." (note capital F to match Python's `False` constant)
2: `True` is only returned when the `versioninfo` is *strictly numerically greather than* it was before. See [code below](https://github.com/theupdateframework/tuf/pull/295/files#diff-852078e79350a80482b37e47cae64ed3R1712).